### PR TITLE
Add Python package PTable

### PR DESCRIPTION
**Pull request by jdblischak**
Links: [PyPI](https://pypi.org/project/PTable/), [GitHub](https://github.com/kxxoling/PTable)
cc: @gaow

### CODE REVIEW

**Comment by jdblischak** [comment]
Note that if both PTable and prettytable are installed, it appears that PTable takes precendence (PTable is a fork of prettytable).

```
$ conda create -y -n tables python
$ source activate tables
$ pip install prettytable PTable
$ conda list | grep '<pip>'
prettytable               0.7.2                     <pip>
PTable                    0.9.2                     <pip>
$ python -c 'import prettytable; print(prettytable.__version__)'
0.9.2
```

**Comment by jdblischak** [comment]
@conda-forge/staged-recipes This Python recipe is ready for review. Thanks!

**Comment by scopatz** [comment]
Thanks @jdblischak - I think that this should require that prettytable is not installed since there is the danger that it would overwrite files.  Also, prettytable should have requirement that Ptable is not installed.

**Comment by jdblischak** [comment]
@isuruf @scopatz Thanks! I never would have figured that out.

To make sure I understand: the primary purpose of `run_constrained` is to specify optional dependencies. By requiring a bogus version number, the recipe effectively prevents the other package from being installed.

I feel better now about not being able to find this! (also this https://github.com/conda/conda-docs/issues/580)

My plan is to send a PR to [prettytable-feedstock]() to add:

```
run_constrained:
   - ptable >=9999
```

File: recipes/ptable/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  home: https://github.com/kxxoling/PTable
+  license: BSD
+  license_family: BSD
+  license_file: COPYING
+  summary: A simple Python library for easily displaying tabular data in a visually appealing ASCII table format
+  doc_url: https://ptable.readthedocs.io
+
+extra:
+  recipe-maintainers:
+    - gaow
```
Comment (scopatz) [inline recipes/ptable/meta.yaml]: @gaow, are you OK being a maintainer!

**Comment by scopatz** [comment]
I also never remember how to do this :)

File: recipes/ptable/meta.yaml (hunk 1 above)
Comment (gaow) [reply recipes/ptable/meta.yaml]: @scopatz yes gladly. Thanks @jdblischak

**Comment by jdblischak** [comment]
Corresponding PR opened for prettytable https://github.com/conda-forge/prettytable-feedstock/pull/13

**Comment by jdblischak** [comment]
> There was discussion about adding a “conflicts” feature to Conda. I don’t recall whether that happened or not. Unfortunately on mobile so can’t link the issues. They should be easy to find on the Conda issue tracker though.

@jakirkham Thanks for the context. I did a quick search. Maybe this Issue: https://github.com/conda/conda/issues/5069

**Pull request merged**